src/controllers/__tests__/messagesController.test.ts

```diff
@@ -339,32 +396,43 @@ describe("MessagesController#getThirdPartyAttachment", () => {
   it("should call the getThirdPartyAttachment on the messagesController with valid values", async () => {
     const req = mockReq();
 
+    // TODO: Define a const with aMessageWithThirdPartyData with values
```

Comment:
Did in [Remove TODO](https://github.com/pagopa/io-backend/pull/1021/commits/80034ad66c058591694f647f849cb1c60c3f4837)

Reply:
I left the comment not to just remove the todo, but to do it :)

Reply:
[Add a message response in test instead of an empty object](https://github.com/pagopa/io-backend/pull/1021/commits/7a858812bec479e40dc57b3ab2749cbf362574d6)